psi4/src/psi4/libmints/extern.h

```diff
@@ -73,6 +73,17 @@ class PSI_API ExternalPotential {
 
     /// Add a charge Z at (x,y,z)
     void addCharge(double Z, double x, double y, double z);
+
+    /// get the vector of charges
+    std::vector<std::tuple<double, double, double, double>> getCharges() const {
```

Comment:
This is C++, so mark `const` and `ref` where possible.

Reply:
Fixed.